core/src/network/event.rs

```diff
@@ -100,9 +100,9 @@ where
     ConnectionEstablished {
         /// The newly established connection.
         connection: EstablishedConnection<'a, TInEvent>,
-        /// The total number of established connections to the same peer,
-        /// including the one that has just been opened.
-        num_established: NonZeroU32,
+        /// The ids of the open connections for the peer, not including the one that has just been
+        /// opened.
+        established_ids: Vec<ConnectionId>,
```

Comment:
```suggestion
        /// List of other connections to the same peer.
        /// 
        /// Note: Does not include the connection reported through this event.
        other_established_connection_ids: Vec<ConnectionId>,
```

I would like to be explicit about the fact that this does not include the currently reported connection, thus the `other_`. What do you think?